felt/src/bigint_felt.rs

```diff
@@ -483,6 +483,9 @@ impl<const PH: u128, const PL: u128> Div for FeltBigInt<PH, PL> {
     // In Felts `x / y` needs to be expressed as `x * y^-1`
     #[allow(clippy::suspicious_arithmetic_impl)]
     fn div(self, rhs: Self) -> Self::Output {
+        if rhs.to_i8().unwrap() == 0 {
+            panic!("Can't divide Felt by zero")
+        }
```

Comment:
Should't we apply this same logic to the other `Div` implementations below this one?
(lines 499 and 514)

Reply:
Also, it would be useful to have some documentation for these implementations indicating possible panics

Reply:
Use `is_zero()` here. Otherwise it will crash with valid inputs (`> 127`).

Reply:
I agree with everything @fmoletta said. We should implement `checked_div` as well so users have the option of handling the error instead, too.

Reply:
https://github.com/lambdaclass/cairo-rs/pull/829/commits/97ffb059e3f750830436c3eab8161513eab193d9 should resolve all this